workspaces/core/src/select/Select.tsx

```diff
@@ -0,0 +1,150 @@
+import React, {forwardRef, useState} from "react";
+import ChevronDown from "../private/icons/ChevronDown";
+import ChevronUp from "../private/icons/ChevronUp";
+import Clear from "../private/icons/Clear";
+import {determineMenuBackgroundColor} from "./private/getBackgroundColor";
+import {getWrapperClasses} from "./private/getWrapperClasses";
+import Option from "./private/types/OptionProps";
+import SelectProps from "./private/types/SelectProps";
+import useEventListeners from "./private/useEventListeners";
+
+const Select = forwardRef<HTMLDivElement, SelectProps>(({
+  options,
+  label,
+  value,
+  disabled,
+  isError,
+  placeholderSlot,
+  placeholderValue,
+  hintSlot,
+  onChange,
+  size = 'md',
+  amountOfVisibleItems = 9999,
+  menuWidth,
+  headerSlot,
+  footerSlot,
+  leftSlot,
```

Comment:
@yarema184  Are you sure because if I do so, component will not be backwards compatible anymore? Previous select component had these props with `slot` suffixes